### PR TITLE
Retain build-status in metadata if defined

### PR DESCRIPTION
**Pull request by gatoravi**
Annotation builds with status "Dummy" prevent builds being launched on the SGMS. This PR retains the build's current status.

### CODE REVIEW

File: lib/perl/Genome/Model/Command/Export/Metadata.pm [hunk 1]
```diff
@@ -341,7 +341,7 @@ sub add_to_dump_queue {
     }
 
     if ($obj->isa("Genome::Model::Build")) {
-        $obj->{status} = "Dummy";
+        $obj->{status} = $obj->status || "Dummy";
```
Comment (tmooney) [inline lib/perl/Genome/Model/Command/Export/Metadata.pm]: `$obj->{status}` should already hold the current value of `$obj->status`, so it should be sufficient to remove this line and the immediately following one that changes `{db_committed}`.

**Comment by gatoravi** [comment]
it seems that a check for a valid status would be appropriate here. A commit to do that has been added.

**Comment by tmooney** [comment]
+1 :ballot_box_with_check:

**Comment by gatoravi** [comment]
merging early since this is relatively minor and would help with SGMS issues.

**Pull request merged**